src/components/Home.jsx

```diff
@@ -30,7 +29,6 @@ const Home = ({ setAppsReady, wrapper }) => {
         <ScrollToTopOnMount target={wrapper} />
         {pathname === '/connected' && <Announcements />}
         <AssistantWrapperDesktop />
```

Comment:
Why are `AssistantWrapperDesktop` and `AssistantWrapperMobile` always rendered? Maybe it should be conditional with an `isMobile`. This logic could be placed in a component called `AssistantWrapper`.

Reply:
fait dans un nouveau commit